packages/test/test-service-load/src/FileLogger.ts

```diff
@@ -93,7 +93,14 @@ class FileLogger implements ITelemetryBufferedLogger {
 		) {
 			event.category = "generic";
 		}
-		this.baseLogger?.send({ ...event, hostName: pkgName, testVersion: pkgVersion });
+		this.baseLogger?.send({
+			...event,
+			hostName: pkgName,
+			testVersion: pkgVersion,
+			details: JSON.stringify({
```

Comment:
How about creating an object for the event in its current form, then `if (process.env.FLUID_TEST_PIPELINE_IDENTIFIER !== undefined) { // Add details property with the desired contents }`, and then calling send()? So we don't add an empty property to the payload.